shared/src/models/material.ts

```diff
@@ -51,7 +53,13 @@ export class Material {
     @Max(maxTreatmentRange)
     public readonly treatmentRange: number;
 
+    // TODO: Create a real Validator.
+    // This is a temporary fix since a Validator of some sort is needed apparently
+    @ValidateIf(() => true)
```

Comment:
Use `@Allow()` for instead.

In `shared\src\store\validation-options.ts` are the default options for class-validator. There you can see that `whitelist: true`.

```
If set to true validator will strip validated object of any properties that do not have any decorators.

Tip: if no other decorator is suitable for your property use @Allow decorator.
```

Reply:
Same for the other such cases.

Reply:
Done